performance/README.md

```diff
@@ -25,18 +25,34 @@ The default base url is `http://localhost:3447` and `fhir/Patient` for the path.
 
 ## Volume Tests
 
-Test the throughput of Hearth when handlng large volumes. Substitute the base url and the path and run:
+Test the throughput of Hearth when handlng large volumes. Substitute the `<BASE_URL>`and the `<PATH>` and run:
```

Comment:
The environment variable for test URI is `<RESOURCE_PATH>`. 
`<PATH>` is an environment variable used by system.